environments/dev/files/archive-requirements.txt

```diff
@@ -2,9 +2,9 @@
 -e git+https://github.com/Connexions/rhaptos.cnxmlutils.git#egg=rhaptos.cnxmlutils
 -e git+https://github.com/Connexions/cnx-epub.git#egg=cnx-epub
 -e git+https://github.com/Connexions/cnx-archive.git#egg=cnx-archive
+-e git+https://github.com/Connexions/cnx-db.git#egg=cnx-db
 
 db-migrator==0.1.4
-cnx-db>=0.2
```

Comment:
Should i leave this, too so the file has both the minimum req. and the git repo lines? That way, the diff w/ production should be just adding repo lines to the top.